src/DependencyInjection/Configuration.php

```diff
@@ -168,4 +181,32 @@ private function resolvePath($value)
 
         return $path;
     }
+
+    /**
+     * Returns the Contao locales.
+     *
+     * @return array
+     */
+    private function getLocales()
+    {
+        $dirs = [__DIR__.'/../Resources/contao/languages'];
+
+        // app/Resources/contao/languages
+        if (is_dir($this->rootDir.'/Resources/contao/languages')) {
+            $dirs[] = $this->rootDir.'/Resources/contao/languages';
+        }
+
+        // The default locale must be the first supported language (see contao/core#6533)
+        $languages = [$this->defaultLocale];
+
+        $finder = Finder::create()->directories()->depth(0)->in($dirs);
+
+        foreach ($finder as $file) {
+            if (preg_match('/^[a-z]{2}(_[A-Z]{2})?$/', $file->getFilename())) {
```

Comment:
Can't the `$finder` do that already?  Like so:

```php
$finder = Finder::create()->directories()->depth(0)->in($dirs)->name('/^[a-z]{2}(_[A-Z]{2})?$/');
```

Reply:
Changed in 4f3558916a72d4a99603b81c3223c9fd82b621e2.